App.js

```diff
@@ -13,6 +14,13 @@ import Entry from './app/Entry';
 import { FlagsProvider } from './app/helpers/Flags';
 import VersionCheckService from './app/services/VersionCheckService';
 
+Sentry.init({
+  dsn:
+    'https://6d77818b863e4e409a0985e64566781f@o416827.ingest.sentry.io/5313567',
```

Comment:
Do you think is right to have this "protocol, public key, the server address, and the project identifier" publicly in the app?

Reply:
Addressed